### PR TITLE
Slider branch - everything that makes the component look real!

### PR DESCRIPTION
**Pull request by Kestrad**

### CODE REVIEW

File: client/components/SimilarListings.jsx [hunk 1]
```diff
@@ -33,12 +34,33 @@ class SimilarListings extends React.Component {
...
+    });
+  }
+
   render() {
     return (
       <div className="similar">
-        {this.state.listings.map(listing => (
-          <Listing listing={listing} />
-        ))}
+        <h1>Similar listings</h1>
```
Comment (Quanquest) [inline client/components/SimilarListings.jsx:56]: Just pointing out semantics: Your index.html is titled "More like this", but you have "Similar listings" as the h1 here.

File: client/components/SimilarListings.jsx [hunk 2]
```diff
@@ -33,12 +34,33 @@ class SimilarListings extends React.Component {
     }
   }
 
+  nextSlide() {
+    this.setState({
+      listings: this.state.listings,
+      position: this.state.position + 1,
+      transform: -100*(this.state.position+1),
```
Comment (Quanquest) [inline client/components/SimilarListings.jsx:41]: I don't know much about transform, so I'm that it doesn't need an initial state.

**Review by Quanquest — APPROVED** [review]
As previously discussed, you've done well with naming conventions on your CSS file to avoid having to work with CSS Modules.

File: client/components/SimilarListings.jsx (hunk 1 above)
Comment (Kestrad) [reply client/components/SimilarListings.jsx:56]: You're right, I will go change the index.html in my next branch.

File: client/components/SimilarListings.jsx (hunk 2 above)
Comment (Kestrad) [reply client/components/SimilarListings.jsx:41]: It doesn't seem to need it? And in fact, the transition property actually listens for a change in the transform property, and coming into existence counts as a change!

**Pull request merged**